apps/docs/src/app/packages/nextwp/core/components/page.mdx

```diff
@@ -65,8 +65,8 @@ export { generateMetadata, generateStaticParams } from "@nextwp/core";
 const templates = {
   page: {
     default: DefaultPageTemplate,
-    home: HomePageTemplate,
-    contact: ContactPageTemplate,
+    homeTemplate: HomePageTemplate,
```

Comment:
I think we don't need this template name part but we could merge the other change!